lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/fraudCri/ContractTest.java

```diff
@@ -0,0 +1,927 @@
+                                                .get("vc")
+                                                .get("credentialSubject");
+
+                                JsonNode vc =
+                                        objectMapper
+                                                .readTree(credential.getJWTClaimsSet().toString())
+                                                .get("vc");
+
+                                JsonNode evidence = vc.get("evidence").get(0);
+
+                                JsonNode nameParts =
+                                        credentialSubject.get("name").get(0).get("nameParts");
+                                JsonNode birthDateNode = credentialSubject.get("birthDate").get(0);
+                                JsonNode addressNode = credentialSubject.get("address").get(0);
+
+                                assertEquals("1", evidence.get("identityFraudScore").asText());
+
+                                assertEquals("GivenName", nameParts.get(0).get("type").asText());
+                                assertEquals("FamilyName", nameParts.get(1).get("type").asText());
+                                assertEquals("Kenneth", nameParts.get(0).get("value").asText());
+                                assertEquals("Decerqueira", nameParts.get(1).get("value").asText());
+
+                                assertEquals("HADLEY ROAD", addressNode.get("streetName").asText());
+                                assertEquals("CURRENT", addressNode.get("addressType").asText());
+                                assertEquals("BA2 5AA", addressNode.get("postalCode").asText());
+                                assertEquals("8", addressNode.get("buildingNumber").asText());
+                                assertEquals("BATH", addressNode.get("addressLocality").asText());
+
+                                assertEquals("1965-07-08", birthDateNode.get("value").asText());
+                            } catch (VerifiableCredentialException
+                                    | ParseException
+                                    | JsonProcessingException e) {
+                                throw new RuntimeException(e);
+                            }
+                        });
+    }
+
+    @Pact(provider = "FraudCriProvider", consumer = "IpvCoreBack")
+    public RequestResponsePact validRequestReturnsExperianFraudCheckResponseWithCi(
+            PactDslWithProvider builder) {
+        return builder.given("dummyApiKey is a valid api key")
+                .given("dummyAccessToken is a valid access token")
+                .given("test-subject is a valid subject")
+                .given("dummyFraudComponentId is a valid issuer")
+                .given("VC is for Kenneth Decerqueira")
+                .given("VC birthDate is 1965-07-08")
+                .given("VC evidence identityFraudScore is 1")
+                .given("VC has CI of A01")
```

Comment:
The VC data has a CI of `P02`, let's go with `A01` as a dummy value unless/until the fraud CRI team say they can't use a mock value.